benchmarks/fastbril-turnt.toml

```diff
@@ -0,0 +1,4 @@
+# command = "bril2json < {filename} | ../build/fastbrili -p {args}"
+command = "bril2json < {filename} | $FASTBRILI -ni -bo byte-out ; cat byte-out | $FASTBRILI -b {args} ; rm byte-out"
```

Comment:
Does this need to be it's own toml file compared to being added to `benchmarks/turnt.toml`?

Reply:
no i just forgot you can put multiple things in one toml file :facepalm: